orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/ContextParameterProcessor.groovy

```diff
@@ -140,15 +141,28 @@ abstract class ContextStringUtilities {
   static String alphanumerical(String str) {
     str.replaceAll('[^A-Za-z0-9]', '')
   }
+
   static String toJson(Object o) {
     new ObjectMapper().writeValueAsString(o)
   }
+
   static Integer toInt(String str) {
     Integer.valueOf(str)
   }
+
   static Float toFloat(String str) {
     Float.valueOf(str)
   }
+
+  static Object fromUrl(String url) {
+    String response = new URL(url).text
+    readJson(response)
```

Comment:
would it make sense to just register the readJson method and enable somebody do do a fromUrl on something that just returned a string? e.g. `https://my.jenkins.net/job/MY-JOB/lastSuccessfulBuild/buildNumber`